README.md

```diff
@@ -72,7 +83,13 @@ Now, the user can load the data `input_data` (e.g. `pandas` `DataFrame`) and res
 3        2021           1  [2021-11-30, 2022-05-29)      460.5   False
 ```
 
-Depending on data preparations, we can choose different types of calendars e.g. [`MonthlyCalendar`](https://ai4s2s.readthedocs.io/en/latest/autoapi/lilio/time/index.html#lilio.MonthlyCalendar) and [`WeeklyCalendar`](https://ai4s2s.readthedocs.io/en/latest/autoapi/lilio/time/index.html#lilio.WeeklyCalendar).
+For convenience, Lilio offers a few shorthands for standard of calendars e.g.
+[`monthly_calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar_shorthands/index.html#lilio.calendar_shorthands.monthly_calendar)
+and
+[`weekly_calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar_shorthands/index.html#lilio.calendar_shorthands.weekly_calendar).
+However, you can also create very complex calendars by calling
+[`Calendar`](https://lilio.readthedocs.io/en/latest/autoapi/lilio/calendar/index.html#lilio.calendar.Calendar)
+directly.
```

Comment:
Perhaps we should refer to the `all_about_the_calendar` notebook here as well (as that feels more explanatory than the docstring).

Reply:
Good point! It would be nice to include these example notebooks in the documentation.

Reply:
Updated the PR with a link to the repo for now.

Reply:
Opened https://github.com/AI4S2S/lilio/issues/23